sdk/apis/apis/v1alpha2/types_apiexport_conversion.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"encoding/json"
+	"fmt"
+
+	kubeconversion "k8s.io/apimachinery/pkg/conversion"
+
+	apisv1alpha1 "github.com/kcp-dev/kcp/sdk/apis/apis/v1alpha1"
+)
+
+const (
+	resourceSchemasAnnotation = "apis.v1alpha2.kcp.io/resource-schemas"
+)
+
+func Convert_v1alpha2_APIExport_To_v1alpha1_APIExport(in *APIExport, out *apisv1alpha1.APIExport, s kubeconversion.Scope) error {
+	out.ObjectMeta = in.ObjectMeta
+
+	// before converting the spec, figure out which ResourceSchemas could not be represented in v1alpha1 and
+	// retain them via an annotation
```

Comment:
surprising. Which are that?

Reply:
ic, this is in preparation of virtual resources? But does not yet apply?

Reply:
yes, this is for one which we don't have "where to put in old API"